doc/src/bond_java.md

```diff
@@ -24,25 +24,30 @@ Basic example
 In Bond data schemas are defined using idl-like
 [syntax](compiler.html#idl-syntax):
 
+```
     namespace examples
```

Comment:
Using both \`\`\` and four spaces causes the code blocks all be indented an extra four spaces when the generated HTML is produced by Pandoc.